mods/ts/sequences/infantry.yaml

```diff
@@ -848,7 +846,6 @@ chamspy:
 	die6: electro
 		Length: *
 		Tick: 80
-	icon: chamicon
```

Comment:
In the original game chamspy is not buildable and the icon is instead used by ghost stalker.